charms/worker/k8s/src/token_distributor.py

```diff
@@ -90,10 +103,10 @@ def revoke(self, name: str, ignore_errors: bool):
             name (str): The name of the node.
             ignore_errors (bool): Whether or not errors can be ignored
         """
-        ...
+        ...  # pylint: disable=unnecessary-ellipsis
 
 
-class ClusterTokenManager(TokenManager):
+class ClusterTokenManager:
```

Comment:
Walks and Talks like a TokenManager, but isn't explicitly one... that's duck-typing

Reply:
Forget about TokenManger -- we don't need 30 lines of typing boilerplate here...